torch/_refs/__init__.py

```diff
@@ -637,6 +651,12 @@ def _ref(
         )
         a, b = _convert_dtype(a, b, dtype=computation_dtype)
 
+        # Special case CPU scalar tensors to be eligible for device transfer
+        if isinstance(a, TensorLike) and a.device.type == 'cpu' and len(a.shape) == 0:
+            a = a.item()
```

Comment:
lol we are having thoughtful discussion about where and if to wrap scalar args, and here turns out we need to unwrap them.

Reply:
Unfortunately it doesn't work lol

```
  File "/private/home/ezyang/pytorch-tmp/torch/_refs/__init__.py", line 663, in _ref
    result = prim(a, b)
  File "/private/home/ezyang/pytorch-tmp/torch/_prims/__init__.py", line 174, in _prim
    return impl_aten(*args, **kwargs)
TypeError: nextafter(): argument 'input' (position 1) must be Tensor, not float
```